.github/workflows/tests.yml

```diff
@@ -149,9 +147,6 @@ jobs:
       matrix:
         config:
           - {os: windows-latest, python: 3.7, ffmpeg: "4.3"}
```

Comment:
I've added FFmpeg 4.4 to the matrix in #997, and that PR now uses our own pyav-ffmpeg library for testing. But if we want to test on Windows against FFmpeg 5.0, we can use the same method with a third-party build at least until our own 5.x builds are complete.

Reply:
Awesome I've merged your PR and rebased this one